test_unstructured_ingest/unit/test_interfaces.py

```diff
@@ -255,3 +258,24 @@ def test_process_file_flatten_metadata(mocker, partition_test_results):
     expected_keys = {"element_id", "text", "type", "filename", "file_directory", "filetype"}
     for elem in isd_elems:
         assert expected_keys == set(elem.keys())
+
+
+class DescribeChunkingConfig:
+    """Unit tests for unstructured.ingest.interfaces.ChunkingConfig"""
+
+    def it_accepts_chunking_strategy_by_itself(self):
+        config = ChunkingConfig(chunking_strategy="basic")
+        assert config.chunking_strategy == "basic"
```

Comment:
Btw, I usually make an exception to the "blank-line-above-and-below" convention for short ones line this where the Act and Assert steps are combined. So I write them just as you have here, although I probably would have tried to fit this one all on one line and avoid the `config` local variable, but that's just me :)